backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/migration/MigrationPolicy.java

```diff
@@ -21,6 +21,8 @@ public class MigrationPolicy implements Serializable, Nameable {
 
     private boolean enableGuestEvents;
 
+    private boolean zerocopy;
```

Comment:
Wouldn't `zeroCopyEnabled` be a better name?

Reply:
Or `zerocopyMigration`.

Reply:
There are already boolean fields with these names: `autoConvergence`, `migrationCompression` and `enableGuestEvents`, each of them using a different naming style. Lets not introduce another :-) I would go either with `zerocopy` (why not `zeroCopy`?) or `enableZerocopy` (or `enableZeroCopy`).

Reply:
It's the same as `autoConvergence` and `migrationCompression`, so "enable" doesn't fit there.  But I don't really care, if you can agree on a different name, I'll change it to it, otherwise I leave it as it is.

> why not zeroCopy?

libvirt uses a single word in most of its symbols and in the command line option.  And I find the single-word version a bit less unreadable in the camelCaseHell.

Reply:
Yeah, seeing it all lower case is hurting my Java eyes :-) Personal opinion aside, lets keep it consistent: all other fields use camel case - lets use it with zero copy too. I think `zeroCopy` would be a good name.

Reply:
Done.